subprojects/docs/docs.gradle

```diff
@@ -123,7 +120,7 @@ outputs.docs = files(docsDir) {
     builtBy 'javadocAll', 'userguide', 'dslHtml', 'releaseNotes'
 }
 
-tasks.withType(AsciidoctorTask) {
+tasks.withType(CacheableAsciidoctorTask) {
```

Comment:
We also need to add a path sensitivity to this file line: 
```
inputs.file asciidocSanitizeStylesheet withPropertyName "sanitizerStylesheet"
```
I guess `NONE`.

Reply:
Done.

Also tested it now with two repository clones in different locations.

Reply:
For a full test, you would also need to use a different Gradle user home directory to have the dependencies in some other place.